aurorawatch-pipeline/test_aurora_transform.py

```diff
@@ -0,0 +1,83 @@
+'''Test file for the AuroraWatch extract script.'''
+# pylint: disable=W0212
+
+from unittest.mock import patch
+from datetime import datetime as dt
+
+import pytest
+
+from aurora_transform import get_colour_id, get_current_datetime, transform
+
+
+class TestGetColourID():
+    '''Tests for the get colour id function.'''
+
+    @pytest.mark.parametrize('colour', [('green'), ('   green  '),
```

Comment:
Good use of parameterisation for testing Aurora colour indicators in a variety of weirder formats.